app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map(function() {
   });
   this.route('login');
   this.route('index', { path: '/' });
+  this.route('waiting-list');
```

Comment:
I could see it being under a particular Artist in the future, maybe, but as of right now, everything is just single-artist, so this feels right to me right now.